serverless-functions/src/functions/common/twilio-wrappers/sync.private.js

```diff
@@ -233,3 +233,84 @@ exports.updateDocumentData = async function updateDocumentData(parameters) {
     return retryHandler(error, parameters, exports.updateDocumentData);
   }
 };
+
+/**
+ * @param {object} parameters the parameters for the function
+ * @param {number} parameters.attempts the number of retry attempts performed
+ * @param {object} parameters.context the context from calling lambda function
+ * @param {string} parameters.name the sid of the Sync stream 
+ * @returns {object} A Sync document
+ * @description the following method is used to fetch a sync document
+ */
+exports.createStream = async function createStream(parameters) {
+  const { context, name } = parameters;
+
+  if (!isObject(context)) throw new Error('Invalid parameters object passed. Parameters must contain context object');
+  if (!isString(name))
+    throw new Error('Invalid parameters object passed. Parameters must contain name of stream string value');
+
+  try {
+    const client = context.getTwilioClient();
+
+    const stream = await client.sync.v1.services(context.TWILIO_FLEX_SYNC_SID).syncStreams.create({ 'uniqueName': name, 'ttl': 86400 });
```

Comment:
Because this is a common function,  we should not set the TTL explicitly, rather allow this to be passed in as a parameter